File: app/src/main/java/bapspatil/flickoff/model/TMDBResponse.java
package bapspatil.flickoff.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by bapspatil
 */

public class TMDBResponse {
    @SerializedName("page") private int page;
    @SerializedName("results") private ArrayList<Movie> results;
    @SerializedName("total_results") private int totalResults;
    @SerializedName("total_pages") private int totalPages;

    public int getPage() {
        return page;
    }

    public void setPage(int page) {
        this.page = page;
    }

    public ArrayList<Movie> getResults() {
        return results;
    }

    public void setResults(ArrayList<Movie> results) {
        this.results = results;
    }

    public int getTotalResults() {
        return totalResults;
    }

    public void setTotalResults(int totalResults) {
        this.totalResults = totalResults;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(int totalPages) {
        this.totalPages = totalPages;
    }
}

File: app/src/main/java/bapspatil/flickoff/model/TMDBResponse.kt
package bapspatil.flickoff.model

import android.annotation.SuppressLint
import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.util.*

/**
 * Created by bapspatil
 */

@SuppressLint("ParcelCreator")
@Parcelize
data class TMDBResponse(
        @SerializedName("page") var page: Int,
        @SerializedName("results") var results: ArrayList<Movie>,
        @SerializedName("total_results") var totalResults: Int,
        @SerializedName("total_pages") var totalPages: Int)
    : Parcelable